themes/openy_themes/openy_lily/sass/modules/_geysir.scss

```diff
@@ -0,0 +1,71 @@
+// Modal window styles.
```

Comment:
if the file is same for all 3 themes, let's keep it on global level via include, like we do for https://github.com/ymcatwincities/openy/blob/8.x-2.x/themes/openy_themes/scss/modules.scss

Reply:
the same for 95%, but has some differences, compare .ui-dialog-titlebar-close style for carnation and rose for example.